### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by virtualguy**
A couple of simple improvements to the documentation found while installing using apache

### CODE REVIEW

File: settings.json.sample [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-    "API_URL": "localhost:5417", # The API URL
-    "SECURE_HTTP": false, # Is the API protected by HTTPS?
-    "templates": { # Templates sample
-        "basic": { # Template name
-            "description": "Basic template", # Template description
+    "API_URL": "http://url.to.salt.api:5417",
```
Comment (Lothiraldan) [inline settings.json.sample]: Please don't include the http protocol, it's override by the SECURE_HTTP config key. I'm not sure if we should change the API_URL value.

For the comments, they are here for pushing people to customize the settings file, I've made recent changes (not yet pushed) to better report errors on the interface when comments were not removed.

**Comment by Lothiraldan** [comment]
Hi @virtualguy, I'm merged some other PRs which fixes the README too, could you update this one to includes only the apache configuration, please? It's easier to merge small PR with one fix.

**Comment by beardedeagle** [comment]
if they don't you may be able to just do that yourself:
```bash
git fetch {remote}
git checkout FETCH_HEAD -- {file}
```

**Comment by virtualguy** [comment]
I've re-written the apache example to better fit the rest of the readme, see #121

**Pull request closed without merging**